hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -354,8 +354,8 @@ public Response createChangeset(@Context HttpServletRequest request,
 
         Map<String, Object> jobStatusTags = new HashMap<>();
         jobStatusTags.put("bbox", reqParams.getBounds());
-        jobStatusTags.put("input1", input1);
-        jobStatusTags.put("input2", input2);
+        jobStatusTags.put("input1", new File(input1).getName());
+        jobStatusTags.put("input2", new File(input2).getName());
```

Comment:
issue was this was storing entire file path instead of the layer id. if it is a name its fine because that means that the file will be uploaded and there wont be an option to do upload changeset from the jobs panel anyways